README.md

```diff
@@ -169,7 +169,7 @@ A holesky integration test can be ran using `make holesky-test` to assert proper
 
 ### Optimism
 
-An E2E test exists which spins up a local OP sequencer instance using the [op-e2e](https://github.com/ethereum-optimism/optimism/tree/develop/op-e2e) framework for asserting correct interaction behaviors with batch submission and state derivation. These tests can be ran via `make optimism-test`.
```

Comment:
Removing this because it no longer exists in the Makefile.
Has it been changed to something else? There's `make e2e-test` but not sure its op specific..